app/frontend/src/components/NotificationBar/_NotificationBar_Close.scss

```diff
@@ -0,0 +1,97 @@
+// External Imports
+@use "sass:map";
+
+// Internal Imports
+@use "../Basics" as *;
+
+/*****************************
+*** Notification Bar class ***
+*****************************/
+$themes: "primary", "success", "warning", "danger";
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+.notification-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
I think we can substitute these for classes in alignments and layout. It won't be a big improvement, though.